packages/editor/src/components/widgets/table/row/MessageRow.css

```diff
@@ -1,9 +1,25 @@
-.table-root .row.row-error {
-  --basic-border: 1px solid var(--error-color);
+.row-warning .table-cell {
+  border-top: var(--error-border) !important;
+  border-bottom: var(--error-border);
 }
```

Comment:
- no important
- table-root .row.row-warning .table-cell
- use warning color for warning row